content/hardware/04.pro/boards/portenta-x8/tutorials/13.wordpress-webserver/content.md

```diff
@@ -64,33 +67,37 @@ services:
   Wordpress:
     depends_on:
       - db
-    image: Wordpress:latest
+    image: wordpress:latest
     volumes:
       - Wordpress_data:/var/www/html
     ports:
       - "8000:80"
     restart: always
     environment:
-      Wordpress_DB_HOST: db
-      Wordpress_DB_USER: Wordpress
-      Wordpress_DB_PASSWORD: Wordpress
-      Wordpress_DB_NAME: Wordpress
+      WORDPRESS_DB_HOST: db
```

Comment:
you sure about this?

Like you are changing the enviromental variables to caps, is this breaking the program or???

Reply:
I was wondering about this too. And from what I was able to replicate a successful result, it is working. So I think there is no problem.